bpy_speckle/connector/operations/load_operation.py

```diff
@@ -71,6 +101,14 @@ def load_operation(context: Context) -> None:
         speckle_obj = traversal_item.current
 
         if not hasattr(speckle_obj, "id"):
```

Comment:
Why does this check exist? were you seeing objects with no id?

If it's come from the server, we can assume it has an id